### PR TITLE
Docs: Add header, logo and initial instructions.

### PR DESCRIPTION
**Pull request by RaulTrombin**
image

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,36 @@
+#![deny(unsafe_code)]
+#![doc(html_logo_url = "https://upload.wikimedia.org/wikipedia/commons/1/12/Bluerobotics-logo.svg")]
+
+/*!
+# Navigator-rs
+
+This crate serves as the entry point for embedding applications using Rust on Blue Robotics's [Navigator](https://bluerobotics.com/store/comm-control-power/control/navigator/).
+
+The Navigator board has the Raspberry Pi HAT form factor, allowing you to attach it to your Raspberry Pi 4 board and unleash the power of your ROV.
```
Comment (patrickelectric) [inline src/lib.rs:9]: Power of Navigator to develop new solutions or interact with the ROV hardware.

Comment (patrickelectric) [reply src/lib.rs:9]: Yes

**Review by joaoantoniocardoso — CHANGES_REQUESTED** [review]
Cool, just a few suggestions

**Comment by RaulTrombin** [comment]
> Cool, just a few suggestions

Thks for your suggestions!

File: src/lib.rs (hunk 1 above)
Comment (RaulTrombin) [reply src/lib.rs]: did it, please check the new suggested aproach.
@patrickelectric

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**